diff --git a/app/src/main/java/com/noahvogt/miniprojekt/messageCreateFragment.java b/app/src/main/java/com/noahvogt/miniprojekt/messageCreateFragment.java
index e36c97e..3afc5e5 100644
--- a/app/src/main/java/com/noahvogt/miniprojekt/messageCreateFragment.java
+++ b/app/src/main/java/com/noahvogt/miniprojekt/messageCreateFragment.java
@@ -6,6 +6,7 @@ import android.content.SharedPreferences;
 
 import android.content.DialogInterface;
 import android.content.Intent;
+import android.net.Uri;
 import android.os.Bundle;
 import android.os.Message;
 import android.text.TextUtils;
@@ -21,6 +22,7 @@ import android.widget.Toast;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.widget.ActivityChooserView;
 import androidx.fragment.app.DialogFragment;
 
 import java.util.concurrent.ExecutorService;
@@ -194,6 +196,11 @@ public class messageCreateFragment extends DialogFragment implements PopupMenu.O
                 if (mailFunctions.validateMessageBody(messageBodyObject) && mailFunctions.validateSubject(subjectObject) &&
                 mailFunctions.validateEmail(receivingAddressObject) && mailFunctions.validateEmail(sendingAddressObject) &&
                 !mailFunctions.checkForSameEmail(sendingAddressObject, receivingAddressObject)) {
+                    /* send email */
+                    Intent intent = new Intent(Intent.ACTION_VIEW,
+                            Uri.parse("mailto:" + receivingAddress));
+                    intent.putExtra(Intent.EXTRA_SUBJECT, subject);
+                    intent.putExtra(Intent.EXTRA_TEXT, messageBody);
                     Toast.makeText(getActivity(), "sending ... ", Toast.LENGTH_SHORT).show();
                     dismiss();
                 } else {
